proto/api/v2/report_service.proto

```diff
@@ -194,6 +207,28 @@ message DeleteReportRequest {
   string id = 1;
 }
 
+message ReportRequestViewBased {
+  enum ReportType {
+    VULNERABILITY = 0;
+  }
+  ReportType type = 1;
+  oneof filter {
+    ViewBasedVulnerabilityReportFilters view_based_vuln_report_filters = 2;
+  }
```

Comment:
Why oneOf if there is only one type of filter?

Reply:
This is done so that ReportRequestViewBased can be reused in future for other reporting features like violations reporting etc